bin/brew

```diff
@@ -46,4 +49,16 @@ fi
 export HOMEBREW_BREW_FILE
 export HOMEBREW_RUBY_PATH
 
-exec "$HOMEBREW_RUBY_PATH" -W0 "$BREW_LIBRARY_DIRECTORY/brew.rb" "$@"
+for i in $@
+do
+  [[ "$i" =~ "^--" ]] || continue
```

Comment:
* We allow flags with one dash, e.g. `-v`. So it should be `"$i" =~ "^-"`
* `||` should be `&&`